src/main/static/stylesheets/variables.scss

```diff
@@ -24,6 +24,8 @@ $width-mobile: 600px;
   --color-highlight-1: #c07cff;
   --color-highlight-2: #9b51e0;
   --color-highlight-3: #791cd0;
+  --color-highlight-4: #efecf4;
```

Comment:
these are both like white, they shouldn't be highlights.

Reply:
also seems like it's being used as a background, use one of the --background variables instead